react/game/src/app/index.js

```diff
@@ -13,16 +14,17 @@ class App extends Component {
   };
 
   loginProps = () => {
```

Comment:
Delete this function. The navigation must be situated in `Login` component. Check `withRouter` [here](https://reacttraining.com/react-router/web/api/withRouter) and [here](https://stackoverflow.com/questions/31079081/programmatically-navigate-using-react-router). U can navigate between routes with `history.push('/new-location')`